screens/FarmListScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import { View, Text } from "react-native";
+import tailwind from "tailwind-react-native-classnames";
+import { useNavigation } from "@react-navigation/native";
+import Screen from "../components/Screen";
+import axios from "axios";
+import FarmItem from "../components/FarmItem";
+
+export default function FarmListScreen() {
+  const navigation = useNavigation();
+
+  const [farmData, setFarmData] = useState();
+
+  const access_token = "1566394169B0EJX2MGAVKVUGGKEMKZBMND9A7VCR";
+
+  const getFarmList = () => {
```

Comment:
How do you think you could improve the UX while the farm data is being fetched from the backend?

Reply:
Using a load or set load to true